### DISCUSSION TITLE
Why does the Stereo SID-player not work?

### DISCUSSION THREAD

**Discussion by ovesennet** [frntc/SIDKick-pico#44] [General]
Hi!
I tried this program: https://commodore.software/downloads/download/47-sid-players/1487-stereo-sid-player-v10-5

It does not play anything when using the SIDKick in dual mode. I have configured it for D500 which works with for example Super mario bros, but not with this program.

It plays, but no audio produced. Any ideas or suggestions to other stereo SID-players that works ?

**Comment by frntc** [comment]
This player crashes on my C64 when loading from SD2IEC :-)

Have you also configured the player to D500?

**Reply by ovesennet** [reply]
Ah ok. I used the Kung Fu Flash cart. Also tried the SD2IEC. Does not work on those.
However, I copied the D64 to a real floppy and now it plays!!

**Comment by ovesennet** [comment]
Closing this as it was not an issue with the SIDKick
